presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,68 @@
+* Free memory on full garbage collection strategy triggers task killer if worker is running low memory and full GC is not able to reclaim enough memory. Low memory threhold is set by  `experimental.task.high-memory-task-killer-heap-memory-threshold`  and Full GC reclaim memory threhold is set by `experimental.task.high-memory-task-killer-reclaim-memory-threshold`.
+* Iceberg connector now supports ANALYZE when configured with a Hive catalog and the table is unpartitioned.
+* Introduces a new flag use_partial_aggregation_history, which controls whether or not partial aggregation histories are used in deciding whether to split aggregates into partial and final.
+* MinbyN/maxbyN's third argument (N) is now checked to be unique.
+* Print information about cost-based optimizers and the source of stats they use (CBO/HBO) in explain plans when session property verbose_optimizer_info_enabled=true.
+* The Iceberg connector now supports `ALTER TABLE <table> ADD COLUMN <column> [WITH (partitioning = '<transform_func>')]` statements to indicate partition transform when add column to table.
+* The Iceberg connector now supports `DELETE FROM <table> [where <filter>]` statements to delete one or more entire partitions.
+* Track null probe keys in join and use it to enable outer join null salt in history based optimization.
+* Update Babel and related npm packages to solve critical vulnerability: https://github.com/advisories/GHSA-67hx-6x53-jw92 The Babel packages are used to generate the JavaScript files for Presto UI. The update prevents the build servers or developers' machines from running arbitrary code while building the JavaScript files.
+* Update Joda-Time to 2.12.5 to use 2023c tzdata. Note: a corresponding update to the Java runtime should also be made to ensure consistent timezone data. For example, Oracle JDK 8u381, tzdata-java-2023c rpm for OpenJDK, or use Timezone Updater Tool to apply 2023c tzdata to existing JVM.
+* Updated docs accordingly (hive connector - schema evolution).
+* When partial aggregation statistics are used in PushPartialAggregationThroughExchange, the optimization also applies to multi-key aggregations (as opposed to single-key only when CBO is used).
+
+Hive Changes
+____________
+* Changes in ParquetPageSourceFactory so that the type-check does not fail with the new widenings supported.
+* Support for Parquet writer versions V1 and V2 is now added for Hive and Iceberg catalogs. It can be toggled using session property `parquet_writer_version` and config property `hive.parquet.writer.version`. Valid values for these properties are PARQUET_1_0 and PARQUET_2_0. Default is PARQUET_2_0. E.g., set session parquet_writer_version=PARQUET_1_0 / hive.parquet.writer.version=PARQUET_1_0.
```

Comment:
`Support Parquet writer versions V1 and V2 in Hive and Iceberg catalogs.`
Let's add a link to the document for the config.

Reply:
@nmahadevuni can you update the hive/iceberg connector docs?